### PR TITLE
Add WikiDocs

### PR DESCRIPTION
**Pull request by bverkron**
Add awesome wikidocs project

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by bverkron** [comment]
Looks like the check failed due to [this](https://github.com/awesome-selfhosted/awesome-selfhosted-data/actions/runs/7358953016/job/20033511055#step:4:80) which is unrelated to my changes?

```shell
ERROR:awesome_lint.py: tine - Community Edition: the project is archived
```

**Comment by nodiscc** [comment]
Hi

> Looks like the check failed due to [this](https://github.com/awesome-selfhosted/awesome-selfhosted-data/actions/runs/7358953016/job/20033511055#step:4:80) which is unrelated to my changes?

correct, your addition passes all automated checks, this failure is unrelated. However,

>  Any software project you are adding was first released more than 4 months ago.

The first [release](https://github.com/Zavy86/WikiDocs/releases) is from 15 hours ago - we must wait until 29 April 2024 before this can be submitted.

**Comment by bverkron** [comment]
Ah I did not realize that the line item is meant to be very explicit about it being a Release in GitHub terms. The developer has been making releases since June 14th (over 6 months ago) but they were not added as an actual Release in Github but rather tags.

https://github.com/Zavy86/WikiDocs/releases/tag/1.0.0

Are there exceptions in cases like this?

**Comment by nodiscc** [comment]
https://github.com/Zavy86/WikiDocs/tags

Tags should be fine.

**Review by kokomo123 — APPROVED** [review]
Seeing as tags should be fine, LGTM. 💯

File: software/wikidocs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "WikiDocs"
+website_url: "http://wikidocs.it"
+source_code_url: "https://github.com/Zavy86/WikiDocs"
+description: "A databaseless markdown flat-file wiki engine."
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Docker
+# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
```
Comment (Ki-er) [inline software/wikidocs.yml:10]: Please remove this line

File: software/wikidocs.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+  - Wikis
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/wikidocs.yml:13]: This line can be removed as its optional

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes

File: software/wikidocs.yml (hunk 2 above)
Comment (bverkron) [reply software/wikidocs.yml]: Oops. I think I went through and removed all of the comments first and then missed that line as being optional when I did further cleanup. Bad workflow approach on my part. Fixed.

File: software/wikidocs.yml (hunk 1 above)
Comment (bverkron) [reply software/wikidocs.yml]: Dang missed it. Sloppy on my part. Fixed.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**